gatsby/src/pages/interactions/introduction.tsx

```diff
@@ -0,0 +1,63 @@
+import { graphql, Link } from "gatsby";
+import { FC } from "react";
+
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { PageHeader } from "@nice-digital/nds-page-header";
+
+import { Layout } from "@/components/Layout/Layout";
+import { SEO } from "@/components/SEO/SEO";
+import { useSiteMetadata } from "@/hooks/useSiteMetadata";
+
+export interface InteractionsIntroductionPageProps {
+	data: {
+		bnfInteractionsIntroduction: {
+			sections: { content: string }[];
```

Comment:
We can probably use `RecordSection` here, see {BnfAboutSection.slug}

Reply:
It differs as it contains an ID property as well as an order property, so I've just specified the fields as part of the new type if that's ok.